sonic-xcvrd/scripts/xcvrd

```diff
@@ -300,8 +308,24 @@ def post_port_dom_info_to_db(logical_port_name, table):
             log_error("This functionality is currently not implemented for this platform")
             sys.exit(3)
 
-# del sfp and dom info from db
-def del_port_sfp_dom_info_to_db(logical_port_name, int_tbl, dom_tbl):
+# update port dom/sfp info in db
+def post_port_sfp_dom_info_to_db():
```

Comment:
rename to "post_port_sfp_dom_info_**from**_db"?

Reply:
@keboliu please have a look at line 328:
# delete port dom/sfp info from db
def del_port_sfp_dom_info_from_db(logical_port_name, int_tbl, dom_tbl):